ADRs/RH/SIG-SRE/ADR-00002-Chaos-Testing-and-Engineering.md

```diff
@@ -0,0 +1,92 @@
+# ADR-00002 - Chaos Testing & Engineering
+
+Authors: Naga Ravi Chaitanya Elluri
+
+
+## Status
+
+Draft
+
+
+## Problem Statement
+
+There are assumptions that users might have when operating and running their applications in distributed systems environment:
+
+The network is reliable.
+There is consistent resource usage with no spikes.
+There is zero latency.
+Bandwidth is infinite.
+The network is secure.
+Topology never changes.
+The network is homogeneous.
```

Comment:
```suggestion
* The network is reliable.
* There is consistent resource usage with no spikes.
* There is zero latency.
* Bandwidth is infinite.
* The network is secure.
* Topology never changes.
* The network is homogeneous.

```

Reply:
The preview showed this as a single paragraph, so converting to a list for easier reading and emphasis on the number of assumptions.